diagnostic.md

```diff
@@ -7,40 +7,41 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
-```
+git clone git@github.com:cpm5041/git-diagnostic.git```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+Install any depencencies if necessary, then change directories
```

Comment:
I think you were on the right track here. After you change into the directory though you want to create a new branch to do your work on.